dydx/dydxViews/dydxViews/_v4/TakeProfitStopLoss/dydxTakeProfitStopLossViewModel.swift

```diff
@@ -115,7 +116,7 @@ public class dydxTakeProfitStopLossViewModel: PlatformViewModel {
                 .themeFont(fontType: .base, fontSize: .small)
             Rectangle()
                 .frame(height: 1)
-                .themeColor(background: .layer6)
+                .overlay(ThemeColor.SemanticColor.layer6.color)
```

Comment:
Does this make a difference?

Reply:
yeah, setting background color on a rectangle doesn't do anything
![Screenshot 2024-04-29 at 7 15 37 PM](https://github.com/dydxprotocol/v4-native-ios/assets/149746839/6742e31b-b6b8-4d07-8e69-f14c91881bf2)